docs/src/components/action/Frame.tsx

```diff
@@ -1,11 +1,10 @@
 import * as React from 'react';
-import { ThemeProvider } from '@material-ui/core/styles';
 import Box, { BoxProps } from '@material-ui/core/Box';
-import { brandingDarkTheme } from 'docs/src/modules/brandingTheme';
 
-const Demo = (props: BoxProps) => {
+const Demo = React.forwardRef<HTMLDivElement, BoxProps>((props, ref) => {
```

Comment:
```suggestion
const Demo = React.forwardRef<HTMLDivElement, BoxProps>(function Demo(props, ref) {
```

please name the render functions of `forwardRef` for a proper display name if devtools.

Issue has probably more instances in this PR.

Reply:
Got it.